### ISSUE TITLE
Not possible to download multiple files (pdf)

### ISSUE THREAD

**Issue by loeffel-io** [chromedp/chromedp#672]
#### What versions are you running?

<pre>
$ go list -m github.com/chromedp/chromedp github.com/chromedp/chromedp v0.5.3
$ google-chrome --version Version 84.0.4147.89 (Offizieller Build) (64-Bit)
$ go version go version go1.14.2 darwin/amd64
</pre>

#### What did you do? Include clear steps.

Logged in @github with `ctx`

```go
ctx2, _ := chromedp.NewContext(ctx)
ctx3, _ := chromedp.NewContext(ctx)

chromedp.Run(ctx2,
	page.SetDownloadBehavior(page.SetDownloadBehaviorBehaviorAllow).WithDownloadPath("."),
	chromedp.Navigate(invoices[0].Href),
)

chromedp.Run(ctx3,
	page.SetDownloadBehavior(page.SetDownloadBehaviorBehaviorAllow).WithDownloadPath("."),
	chromedp.Navigate(invoices[1].Href),
)
```

#### What did you expect to see?

2 PDFs

#### What did you see instead?

Only one PDF - Blocks after downloaded first pdf

**Comment by opennota** [comment]
I can confirm this.

A self-contained example:

``` Go
package main

import (
	"context"
	"fmt"
	"log"
	"net/http"

	"github.com/chromedp/cdproto/page"
	"github.com/chromedp/chromedp"
)

func main() {
	c, _ := chromedp.NewExecAllocator(
		context.Background(),
		chromedp.Flag("headless", false),
		chromedp.NoDefaultBrowserCheck,
		chromedp.NoFirstRun,
	)

	ctx1, _ := chromedp.NewContext(c)
	ctx2, _ := chromedp.NewContext(c)

	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
		w.Header().Set("Content-Disposition", fmt.Sprintf(`attachment; filename="%s"`, r.URL.Path[1:]))
		w.Write([]byte("xyz"))
	})
	go func() {
		log.Fatal(http.ListenAndServe(":8000", nil))
	}()

	if err := chromedp.Run(
		ctx1,
		page.SetDownloadBehavior(page.SetDownloadBehaviorBehaviorAllow).WithDownloadPath("."),
		chromedp.Navigate("http://localhost:8000/1.xyz"),
	); err != nil {
		log.Println(err)
	}

	if err := chromedp.Run(
		ctx2,
		page.SetDownloadBehavior(page.SetDownloadBehaviorBehaviorAllow).WithDownloadPath("."),
		chromedp.Navigate("http://localhost:8000/2.xyz"),
	); err != nil {
		log.Println(err)
	}

	select {}
}
```

**Comment by ZekeLu** [comment]
I just tested @opennota 's code, it works with `chromedp@v0.7.1` and `Chrome 90.0.4430.93`.

BTW, it should listen to the `page.EventDownloadProgress` event to wait for the download to finish.

Anyway, the download command and events in the `page` domain are deprecated. They have been moved to the `browser` domain. Check https://github.com/chromedp/examples/blob/master/download_file/main.go for an example.

Closing this issue because it's a little outdated.
